### PR TITLE
Embed more metadata in exported model and read it in native client

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

File: DeepSpeech.py [hunk 1]
```diff
@@ -724,6 +724,17 @@ def do_graph_freeze(output_file=None, output_node_names=None, variables_blacklis
         if not FLAGS.export_tflite:
             frozen_graph = do_graph_freeze(output_node_names=output_names, variables_blacklist='previous_state_c,previous_state_h')
             frozen_graph.version = int(file_relative_read('GRAPH_VERSION').strip())
+
+            # Add a no-op node to the graph with metadata information to be loaded by the native client
```
Comment (lissyx) [inline DeepSpeech.py]: And we don't break with TF Lite? nice :)

Comment (reuben) [reply DeepSpeech.py]: Nope. We don't get the info in the TFLite graph either, unfortunately. I'll try to find a workaround when I have some time.

File: util/flags.py [hunk 2]
```diff
@@ -73,6 +77,7 @@ def create_flags():
     tf.app.flags.DEFINE_boolean ('export_tflite',    False,       'export a graph ready for TF Lite engine')
     tf.app.flags.DEFINE_boolean ('use_seq_length',   True,        'have sequence_length in the exported graph (will make tfcompile unhappy)')
     tf.app.flags.DEFINE_integer ('n_steps',          16,          'how many timesteps to process at once by the export graph, higher values mean more latency')
+    tf.app.flags.DEFINE_string  ('export_model_language', '',     'language the model was trained on. Gets embedded into exported model.')
```
Comment (lissyx) [inline util/flags.py]: I'd like we get a different naming, it's confusing, I thought it was the language model we were embedding, I ad t ore-read carefully several time. Can the option be named "add_language_metadata" ?

Comment (reuben) [reply util/flags.py]: How about this:

`tf.app.flags.DEFINE_string  ('export_language',  '',          'language the model was trained on e.g. "en" or "English". Gets embedded into exported model.')`

Comment (lissyx) [reply util/flags.py]: WFM, feels less confusing

**Comment by reuben** [comment]
OSX workers are acting up but the relevant code has been exercised in other completed tests. Merged manually to avoid starting another group here in the PR.

**Pull request merged**